src/Components/EmailNotificationForm/EmailNotificationForm.css

```diff
@@ -0,0 +1,73 @@
+.label-input-container, .preferred-lang-p, .form-header {
+  font-size: 20px;
+  color: #2b2d42;
+  display: block;
+}
+
+.state-name-select {
+  font-size: 1.1rem;
```

Comment:
Oh!  It looks like you've used rems for font-sizes, so it's a familiar unit!  Awesome!